test/services/chart_builder_test.rb

```diff
@@ -4,21 +4,63 @@ class ChartBuilderTest < ActiveSupport::TestCase
   test "#build_columns" do
     benchmark_run = create(:commit_benchmark_run, result: { 'some_time' => 5, 'some_other_time' => 5 })
     other_benchmark_run = create(:commit_benchmark_run, result: { 'some_time' => 5, 'some_other_time' => 5 })
-    chart_builder = ChartBuilder.new([benchmark_run, other_benchmark_run])
+    chart_builder = ChartBuilder.new([benchmark_run, other_benchmark_run], benchmark_run.benchmark_result_type)
 
-    chart_columns = chart_builder.build_columns do |benchmark_run|
-      "Commit: #{benchmark_run.initiator.sha1}"
+    chart_builder2 = chart_builder.build_columns do |benchmark_run|
+      { commit: benchmark_run.initiator.sha1 }
     end
 
+    # check that `build_columns` returns self
+    assert_equal chart_builder, chart_builder2
```

Comment:
If you want to compare that objects are the same, it has to be `assert_equal chart_builder.object_id, chart_builder2.object_id`. You also won't need the comment because comparing `object_id` tells developers exactly what the comment is trying to tell